configure.ac

```diff
@@ -500,7 +500,7 @@ AS_IF([test x"$with_cuda" != x"no"],
 			      [])],
 	    [])
 
-AS_IF([test "$with_cuda" = "yes" && test "$have_libcuda" = "0" ],
+AS_IF([test x"$with_cuda" != x"no" && test -n "$with_cuda" && test "$have_libcuda" = "0" ],
```

Comment:
Maybe a silly question, what does "x" mean here?

Reply:
That's to handle tests of strings that could be mistaken for a parameter (starting with a`-`) or an empty string.

Reply:
I see, thanks.